src/collective/cover/tests/test_base_tile.py

```diff
@@ -71,9 +68,14 @@ def test_default_configuration(self):
     def test_accepted_content_types(self):
         self.assertEqual(self.tile.accepted_ct(), ALL_CONTENT_TYPES)
 
-    def test_delete_tile_persistent_data(self):
-        eventtesting.clearEvents()
-
+    def test_delete_tile_removes_persistent_data(self):
+        # https://github.com/collective/collective.cover/issues/765
+        from collective.cover.config import CONFIGURATION_PREFIX
+        from collective.cover.config import PERMISSIONS_PREFIX
+        from zope.annotation import IAnnotations
+        from zope.component import eventtesting
+        from zope.lifecycleevent import IObjectModifiedEvent
```

Comment:
@hvelarde  why in tests do you import inside the methods?

Reply:
because is easier to maintain later.